CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.8.0
+
+* Make `isBrowser` and `isNodeJs` more reliable.
```

Comment:
Can you be more explicit about what circumstances these were unreliable in the past that are now improved?